.github/workflows/runbenchandupload.yml

```diff
@@ -0,0 +1,62 @@
+name: RunUploadBenchmarks
+env:
+  JULIA_NUM_THREADS: 1
+on:
+  pull_request:
+    types: [labeled]
+    branches:
+     - master
+     - benchx
+jobs:
+  benchmark:
+    if: ${{ github.event.label.name == 'to-benchmark' }}
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.allow_failure }}
+    permissions: write-all
```

Comment:
can we be more granular?

Reply:
I don't think this workflow needs any write permission?

Reply:
ah yes. I will remove the `permissions` as a whole then.